src/components/data-pane/data-selector.tsx

```diff
@@ -188,9 +188,10 @@ const DataSelectorRenderer = CSSModules(DataSelectorBase, styles);
 
 export const DataSelector = connect(
   (state: State) => {
+    const presentUndoableState = state.undoable.present;
     return {
-      data: state.present.dataset,
-      name: state.present.dataset.name,
+      data: presentUndoableState.dataset,
+      name: presentUndoableState.dataset.name,
```

Comment:
It's weird that we need name since it's already a part of data.  File an issue and submit a separate PR?